test/rotoscope_test.rb

```diff
@@ -10,11 +10,6 @@
 require 'fixture_inner'
 require 'fixture_outer'
 
-# For some reason an object finalizer is causing CI to exit with status 1
-# without any messages if it happens after minitest calls `exit`.  This will
-# workaround it for now
-Minitest.after_run { GC.start }
```

Comment:
Changing this line to the following would reliably reproduce the exit status error

```ruby
Minitest.after_run { Rotoscope.new('tmp/test2.csv.gz').start_trace }
```